src/test/java/com/johncnstn/auth/constraints/PasswordValidatorTest.java

```diff
@@ -0,0 +1,57 @@
+package com.johncnstn.auth.constraints;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class PasswordValidatorTest {
```

Comment:
Why is it not extending one of abstract classes for tests?

Reply:
Done